src/modules/portfolio/components/transaction-meta/transaction-meta.jsx

```diff
@@ -23,6 +25,10 @@ export default class TransactionMeta extends Component {
     } = this.props
     const baseLink = networkId ? TransactionMeta.networkLink[networkId] : null
 
+    if (typeof meta.timestamp === 'number') {
+      meta.timestamp = formatDate(new Date(meta.timestamp * 1000)).full
```

Comment:
See the convert timestamp function in format-date

Reply:
Good call, fixed!